test/unit-breaking/README.md

```diff
@@ -0,0 +1 @@
+Direcory for unit tests that are activly not working with the current chanes
```

Comment:
Fixed typos & refactored:

`Directory for unit tests that are not currently not working with the current state of the driver. `

Reply:
Why `unit-breaking`? Maybe `unit-broken` or `unit-nonworking`?